File: April/DisemvowelTrolls.py
# Disemvowel Trolls

# CodeWars Link - https://www.codewars.com/kata/52fba66badcd10859f00097e

# Problem Statement
# Remove all of the vowels from the trolls' comments, neutralizing the threat.

# Example
# Input: "This website is for losers LOL!" -> Output: "Ths wbst s fr lsrs LL!"
# Note: y isn't considered a vowel for this post


# My Solution
# Iternate throgh comment -> if the current char matches a vowel, replace with space -> return comment
def disemvowel(comment):

    for char_ in comment[:]:
        if char_ in "aeiouAEIOU":
            comment = comment.replace(char_, '')

    return comment

File: July/DisemvowelTrolls.js
// Disevowel Trolls

//CodeWars Link - https://www.codewars.com/kata/52fba66badcd10859f00097e

// Problem Statement
// Your task is to write a function that takes a string and return a new string with all vowels removed.
// For example, the string "This website is for losers LOL!" would become "Ths wbst s fr lsrs LL!".


// My Solution
function disemvowel(str) {
    return str.replace(/[aeiouAEIOU]/g, "");
}

// Better Solution
// uses gi which is case insensitive
function disemvowel(str) {
    return str.replace(/[aeiou]/gi, "");
}
